### ISSUE TITLE
Attention-grabbing arrow points doesn't always point to a level that makes sense

### ISSUE THREAD

**Issue by phoenixeliot** [codecombat/codecombat#3466]
When using a paid account (with a subscription), the big yellow arrow on the level select screen usually points to the earliest incomplete paid level. The result is that it's a little hard to find the level I want to do next/just unlocked (and sometimes don't notice when multiple new levels are unlocked), and my attention is drawn to and old level I don't feel like doing.

This is not consistent across worlds, though. This occurs in "Forest", but not usually in "Dungeon". I haven't checked further levels.

Needs more checking for what conditions this occurs under. I'm not sure how it interacts with non-paid accounts, though Josh says it often points to paid levels, even on unpaid accounts, making it seem like the user needs to pay to get the next level.

The black-circled levels are ones that would perhaps make more sense to have an arrow pointing at them, but for this paid account and level completeness state, that bonus level is *always* the one that gets the arrow and has been for most of this area.

<img width="1218" alt="screen_shot_2016-03-11_at_10_09_40_am" src="https://cloud.githubusercontent.com/assets/491376/13711077/a4eed33a-e771-11e5-962d-6de0533bf35b.png">

**Comment by nwinter** [comment]
Yeah, it could use some work. Code is here: https://github.com/codecombat/codecombat/blob/master/app/views/play/CampaignView.coffee#L325-L352

We might also do [some of this](https://discourse.codecombat.com/t/easier-revision-suggestion-to-help-with-level-progression/6945/2).

**Comment by themaka** [comment]
It sometimes points to subscriber-only levels for non-subscribers as well (which has some value to us as well ...)
